### PR TITLE
Adding contributor policies

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
A few basic contributor items like code of conducts, issue/PR templates, etc.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+```
+Show here how to reproduce the new behavior (can be a bug fix or a new feature)
+```
+
+## Checklist
+
+- [ ] Have you checked our [Contributing](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/docs/contribute.rst) document?
+- [ ] Have you ensured the PR description clearly describes the problem and solutions?
+- [ ] Have you checked to ensure that there aren't other open or previously closed [Pull Requests](https://github.com/catalystneuro/nwb-conversion-tools/pulls) for the same change?
+- [ ] If this PR fixes an issue, is the first line of the PR description `fix #XXX` where `XXX` is the issue number?
```
Comment (rly) [inline .github/PULL_REQUEST_TEMPLATE.md]: Please update links

Comment (CodyCBakerPhD) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yup, still WIP haha

Comment (CodyCBakerPhD) [reply .github/PULL_REQUEST_TEMPLATE.md]: Was also thinking about splitting contributions into two subtypes (GitHub makes the buttons automatically): "Add a check" and "Add a feature" - the check one would have specific To-Do items for writing checks, things like adding the decorator, values in `InspectorMessage`, etc.

How does that sound?

Comment (rly) [reply .github/PULL_REQUEST_TEMPLATE.md]: I like that idea

**Review by rly — APPROVED** [review]

**Review by bendichter — APPROVED** [review]

**Pull request merged**